tools/recalc.py

```diff
@@ -66,7 +66,6 @@ async def recalculate_score(
     calculator = Calculator(
```

Comment:
can we have a similar check/err raise here?

Reply:
or even better, could we make the recalc.py script use the usecase function?

Reply:
this should probably be using the usecase, no?

Reply:
> an we have a similar check/err raise here?
we don't need one, we simply just don't specify the acc here, why would we check for anything

Reply:
Oh I see, this should be using the usecase yeah

Reply:
Follow-up PR is fine